src/components/NavBar.js

```diff
@@ -0,0 +1,30 @@
+import { Navbar, Nav, Container } from "react-bootstrap";
+import React from "react";
+import navIcon1 from '../assets/img/block-linkedin.png';
+import navIcon2 from '../assets/img/block-github.png';
+import navIcon3 from '../assets/img/block-fb.png';
+import logo from '../assets/img/logo.png';
+
+export const NavBar = () => {
+  return (
+    <Navbar expand="lg">
+      <Container>
+        <Navbar.Brand href="#home">
+          <img src={logo} alt="Logo" className="logo-img"/>
+        </Navbar.Brand>
+        <Navbar.Collapse id="basic-navbar-nav">
+          <Nav className="ms-auto">
+            <Nav.Link href="#home" className='navbar-link-home'></Nav.Link>
```

Comment:
I think it makes more sense to render the buttons of this `NavLink` as image instead of background
since it is not used as a background